.claude/skills/cluster-provision/SKILL.md

```
@@ -0,0 +1,80 @@
+# Cluster Provision
+
+Provision an ephemeral OpenShift cluster from a Hive ClusterPool via the OpenShift CI Gangway REST API.
+
+## Arguments
+
+Parse `$ARGUMENTS` into at most two values before invoking Bash:
+- **Arg 1**: KUBECONFIG path (default: `/tmp/k`)
+- **Arg 2**: OCP version (default: `4.18`)
+
+Set variables from `$ARGUMENTS`:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)